MCprep_addon/spawner/mcmodel.py

```diff
@@ -323,7 +323,7 @@ def read_model(
 	"""
 	try:
 		with open(model_filepath, 'r') as f:
-			obj_data = json.load(f)
+			obj_data = json.load(open(model_filepath, encoding="utf-8-sig"))
```

Comment:
This additional `open` call is unneeded (and could cause issues with resource cleanup), the encoding can be passed to the `open` call in the `with` statement.